softlayer/softlayer_product_package_service.go

```diff
@@ -9,6 +9,8 @@ type SoftLayer_Product_Package_Service interface {
 
 	GetItemPrices(packageId int) ([]datatypes.SoftLayer_Product_Item_Price, error)
 	GetItemPricesBySize(packageId int, size int) ([]datatypes.SoftLayer_Product_Item_Price, error)
+	GetItemPricesBySizeAndIops(packageId int, size int, capacity int) ([]datatypes.SoftLayer_Product_Item_Price, error)
```

Comment:
Hi, @jianqiu, are these service methods in the `SL_Product_Item_Price` service? If yes, then this PR makes sense and should be merged in SL go, if not then let's move these into the SL CPI code. Maybe create them into a file in the `utils` package.  Thoughts?